qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -469,6 +469,11 @@ def to_complex(coeff):
             PauliList.from_symplectic(z, x), coeffs, ignore_pauli_phase=True, copy=False
         )
 
+    def is_zero(self) -> bool:
```

Comment:
I wouldn't like to add the method to `SparsePauliOp`, it's inconsistent in `quantum_info`.
It would be nice to have the utility function `is_zero(op: SparsePauliOp)`.
Or we should add this method to all classes.

Reply:
I added a static `_sparse_pauli_op_is_zero()` method to the Z2Symmetries class and took out the `is_zero` one from SparsePauliOp